backend/Dockerfile.worker

```diff
@@ -27,11 +27,11 @@ RUN go mod init crossfeed-worker
 
 RUN go get github.com/facebookincubator/nvdtools/...@v0.1.4
 
-FROM node:16.19.0-alpine3.17
+FROM node:18-alpine3.17
 
 WORKDIR /app
 
-RUN apk add --update --no-cache wget build-base curl git unzip openssl-dev linux-headers python3=3.10.10-r0 python3-dev py3-pip ruby ruby-dev zlib-dev libffi-dev libxml2-dev libxslt-dev postgresql-dev gcc musl-dev py3-pandas py3-scikit-learn cargo
+RUN apk add --update --no-cache wget build-base curl git unzip openssl-dev linux-headers python3=3.10.11-r0 python3-dev py3-pip ruby ruby-dev zlib-dev libffi-dev libxml2-dev libxslt-dev postgresql-dev gcc musl-dev py3-pandas py3-scikit-learn cargo
```

Comment:
Is it possible that it's failing because many packages do not have wheels already available for Python 3.10.11? (and maybe better to downgrade to a version of python where these wheels are already available?) just thinking...

Reply:
3.10.10 wasn’t in the image, we have to use the latest image otherwise the
build will fail.

Reply:
just added scikit-learn==1.2.2 to work around old package.